templates/jetty/files/README.md

```diff
@@ -0,0 +1,25 @@
+# Eclipse Jetty
+
+Eclipse Jetty provides a Web server and javax.servlet container, plus support for HTTP/2, WebSocket, OSGi, JMX, JNDI, JAAS and many other integrations. These components are open source and available for commercial use and distribution.
+
+Eclipse Jetty is used in a wide variety of projects and products, both in development and production. Jetty can be easily embedded in devices, tools, frameworks, application servers, and clusters.
+
+## Services
+
+* Java 8
+
+## Customizations
+
+The following files and additions make the framework work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
+
+* [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
+* [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
+* [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
+* An additional library dependency, [`platformsh/config-reader-java`](https://github.com/platformsh/config-reader-java), has been added.  It provides convenience wrappers for accessing the Platform.sh environment variables.
+
+## References
+
+* [Platform.sh post](https://platform.sh/blog/2019/java-hello-world-at-platform.sh/)
```

Comment:
Do we need to link to blog posts from the READMEs?  Those feel far too ephemeral.